example/ios/Podfile

```diff
@@ -6,7 +6,11 @@ platform :ios, '11.0'
 target 'Example' do
   config = use_native_modules!
 
-  use_react_native!(:path => config["reactNativePath"])
+  use_react_native!(
```

Comment:
Are all those changes of `example` project necessary in this PR? They seem to not concern it.

Reply:
These changes restore the original Podfile from RN 0.67.2 template so we can build the app on our M1s. Will move them to a separate PR.

Reply:
Done in 58f752099ae1bbf53377258b61db4e9a8bd42a26, in favor of #1903